pkg/gather/gcp/gcp.go

```diff
@@ -59,10 +59,7 @@ func New(logger logrus.FieldLogger, serialLogBundle string, bootstrap string, ma
 
 // Run is the entrypoint to start the gather process.
 func (g *Gather) Run() error {
-	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
-	defer cancel()
-
-	svc, err := compute.NewService(ctx, option.WithCredentials(g.credentials))
+	svc, err := compute.NewService(context.Background(), option.WithCredentials(g.credentials))
```

Comment:
This will give no timeout to service creation. If the service hangs that could force the user into a interrupt or waiting for like 30 minutes. I suppose that is fine for the time being and we can adjust it as needed. 
/approve
/lgtm